Makefile

```diff
@@ -0,0 +1,26 @@
+BUILD_DIR=build
```

Comment:
IMHO, having a "temporary" `bin` directory besides the `$GOBIN` is kinda redundant.

Reply:
Cross-compiling for ARM on build host, for example.